draft-ietf-taps-interface.md

```diff
@@ -1356,9 +1423,11 @@ functionality to implement Clone. In that case, entangled Connections are
 multiplexed together, giving them similar treatment not only inside endpoints
 but also across the end-to-end Internet path.
 
-Note that calling Clone() may result in on-the-wire signaling, e.g., to open a new connection, depending on the underlying Protocol Stack.
+Note that calling Clone() may result in on-the-wire signaling, e.g., to open a new connection, depending on the underlying Protocol Stack. In this case, the transport system will ensure consistency of
+Connection Properties by uniformly applying them to all underlying connections
+in a group.
 
-If the underlying Protocol Stack does not support cloning, or cannot create a
+If the underlying Protocol Stack cannot create a
```

Comment:
Ahh... "fixed" this too fast!  I completely agree that this is also a possible implementation, and hence I wanted to remove "does not support cloning" - there's no reason not to support it. In doing so, I removed too little because indeed now it sounds even "narrower". Thanks!